src/util.test.js

```diff
@@ -114,6 +115,7 @@ describe('convertToSlipData', () => {
     expect(o['requester.firstName']).toEqual(source.requester.firstName);
     expect(o['requester.lastName']).toEqual(source.requester.lastName);
     expect(o['requester.middleName']).toEqual(source.requester.middleName);
+    expect(o['requester.preferredFirstName']).toEqual(source.requester.preferredFirstName);
```

Comment:
Could you please investigate possible add test to RTL/Jest when preferredFirstName are absent?

Reply:
added another unit test